src/View/View.php

```diff
@@ -1244,7 +1244,26 @@ public function helpers(): HelperRegistry
     }
 
     /**
-     * Loads a helper. Delegates to the `HelperRegistry::load()` to load the helper
+     * Adds a helper from within `initialize()` method.
+     *
+     * @param string $helper Helper.
+     * @param array<string, mixed> $config Config.
+     * @return void
+     */
+    protected function addHelper(string $helper, array $config = []): void
+    {
+        [$plugin, $name] = pluginSplit($helper);
+        if ($plugin) {
+            $config['className'] = $helper;
+        }
+
+        $this->helpers[$name] = $config;
+    }
+
+    /**
+     * Loads a helper. Delegates to the `HelperRegistry::load()` to load the helper.
+     *
+     * Deprecated: The use within `initialize()` method is deprecated, use `addHelper()` instead.
```

Comment:
```suggestion
     * You should use `addHelper()` instead of this method from the `initialize()` hook of `AppView` or other custom View classes.
```